### DISCUSSION TITLE
9.0.0 released!

### DISCUSSION THREAD

**Discussion by Feiyang1** [firebase/firebase-js-sdk#5376] [General]
[Release notes](https://firebase.google.com/support/release-notes/js#version_900_-_august_25_2021)

**Comment by jeffcad** [comment]
Already upgraded my work-in-progress Next.js project which uses Firestore. Bundle size is reduced by 36KB. Another work-in-progress Next.js project which uses Firestore and Auth has been reduced by 75KB. Very good.

**Comment by earthiverse** [comment]
I'm brand new to Firebase, trying it out for the first time today.

I'm trying to follow the [Get started with Firebase Hosting](https://firebase.google.com/docs/hosting/quickstart) tutorial, but the page that's generated by `firebase init hosting` errors out with `Error loading the Firebase SDK, check the console.` and a lot of console errors.

```
Uncaught SyntaxError: export declarations may only appear at top level of a module firebase-app.js:1590
Uncaught SyntaxError: import declarations may only appear at top level of a module firebase-auth.js:1
Uncaught SyntaxError: import declarations may only appear at top level of a module firebase-database.js:1
Uncaught SyntaxError: import declarations may only appear at top level of a module firebase-firestore.js:1
Uncaught SyntaxError: import declarations may only appear at top level of a module firebase-functions.js:1
Uncaught SyntaxError: import declarations may only appear at top level of a module firebase-messaging.js:1
Uncaught SyntaxError: import declarations may only appear at top level of a module firebase-storage.js:1
Uncaught SyntaxError: import declarations may only appear at top level of a module firebase-analytics.js:1
Uncaught SyntaxError: import declarations may only appear at top level of a module firebase-remote-config.js:1
Uncaught SyntaxError: import declarations may only appear at top level of a module firebase-performance.js:1
Uncaught Error: hosting/init-error: Firebase SDK not detected. You must include it before /__/firebase/init.js
    <anonymous> https://almerchants.web.app/__/firebase/init.js?useEmulator=true:1
init.js:1:44
ReferenceError: firebase is not defined
    <anonymous> https://almerchants.web.app/:69
    EventListener.handleEvent* https://almerchants.web.app/:51
almerchants.web.app:83:19
```

If I change the versions from `9.0.0` to `8.10.0` it still works...

How do I get version 9 to work?

**Reply by Feiyang1** [reply]
It seems the page generate `firebase init hosting` doesn't work with v9 SDK. We will need to update firebase cli to fix it.
Can you follow the [get started guide](https://firebase.google.com/docs/web/setup) to include firebase SDKs manually?

**Comment by Dixon-obabo** [comment]
This is still an issue up to currently 9.6.6
